cylc/flow/scripts/validate_reinstall.py

```diff
@@ -135,10 +135,16 @@ def vro_cli(parser: COP, options: 'Values', workflow_id: str):
     try:
         detect_old_contact_file(workflow_id)
     except ServiceFileError:
-        # Workflow is definately running:
+        # Workflow is definitely running:
         workflow_running = True
+    except Exception:
```

Comment:
For debugging purposes, don't swallow the original exception
```suggestion
    except Exception as exc:
        LOG.error(exc)
```